contrib/pyln-client/pyln/client/gossmap.py

```diff
@@ -25,33 +25,35 @@
 
 
 class GossipStoreHeader(object):
```

Comment:
It (the change) is not really worse in terms of performance, also I needed the internal variables (`offfset` and `ratelimited`). This is needed for debugging and special flags. The old code was instantiating the Header anyway and throwing it away afterwards.

Currently it can read a whole unpurged 75MByte gossip store (15k nodes and 75k channels) in about 8 seconds.